packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -180,8 +181,71 @@ const SearchContainer = styled.div`
   }
 `;
 
+const SiteContextTitle = styled.div`
+  margin-right: ${designSystem.dimensions.spacings.s};
+`;
+
+const SiteTitle = styled.div`
+  margin-right: ${designSystem.dimensions.spacings.s};
+  ${({ hasContext }) =>
+    hasContext &&
+    `
+    font-weight: 600;
+  `}
+`;
+
+const TitleContainer = styled.div`
+  ${({ hasContext }) =>
```

Comment:
see below, there should rather be a fallback value for UI consistency.  You can simplify the code then.

Reply:
A default value for the actual SiteContext if a given microsite is not appearing explicitly in the top menu